### DISCUSSION TITLE
Storage options for using storage driver as overlay2

### DISCUSSION THREAD

**Discussion by alpana17** [cri-o/cri-o#7979] [General]
Default storage driver with CRI-O is overlay
If I move to overlay2 by setting it in /etc/crio/crio.conf, what should I set as storage _option?

Current output of cmd (crio-status config 2>/dev/null | grep storage) is:
`  root = "/var/lib/containers/storage"
  runroot = "/run/containers/storage"
  storage_driver = "overlay"
  storage_option = ["overlay.mountopt=nodev,metacopy=on"]`

**Comment by alpana17** [comment]
@haircommander please see

**Comment by haircommander** [comment]
in cri-o overlay driver actually means overlay2. so just set the overlay driver and you should be using overlay2

**Reply by alpana17** [reply]
ok
Thanks

**Comment by alpana17** [comment]
@haircommander Is it mentioned somewhere in any document or can I see that in source code?

**Reply by haircommander** [reply]
they're interpreted the same in the config https://github.com/containers/storage/blob/58bffcae18f1a679ceac0e9502f321fc59a275fa/pkg/config/config.go#L290 though I recommend using `overlay` as i've seen issues in the past when users choose overlay2 even though they should be functionally equivalent

**Reply by alpana17** [reply]
ok
Thank you
